app/assets/src/components/views/pipeline_viz.scss

```diff
@@ -28,14 +30,15 @@
 }
 
 .graphLabel {
-  display: "flex";
-  justify-content: "space-between";
-  flex-wrap: "wrap";
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
 }
 
-.closeButton {
+.closeIcon {
   padding: 3px 12px;
-  cursor: pointer;
+  overflow: visible !important;
```

Comment:
Why do you need important?

Reply:
The icon wasn't showing, but it was due to the padding — resolved!